### ISSUE TITLE
Support For TLS Version 1.2

### ISSUE THREAD

**Issue by kler** [auth0/auth0-PHP#479]
From 2021-03-22 Auth0 will be requiring TLS 1.2.

- Does this package support that?
- If yes, from what version is TLS 1.2 supported? _(i.e., who will need to upgrade)_

See
- https://support.auth0.com/notifications/60075b84581057000b4a6739
- https://github.com/auth0/node-auth0/issues/581

**Comment by evansims** [comment]
**TL;DR — Yes, so long as you're using a [supported version](https://www.php.net/supported-versions.php) of PHP.**

This library supports TLS Version 1.2 inherently, so long as the PHP runtime used to execute it is compiled with a version of OpenSSL that supports TLS 1.2 (OpenSSL 1.0.1 or newer.)

As PHP runtimes can be compiled with different configurations of OpenSSL depending on the build environment, there is no single answer I can give you as to what version of PHP began supporting this, but I see references to it back in 5.6 so that is an indicator that has been supported for quite some time.

If you are running a supported version of PHP (7.3+) with OpenSSL support >= 1.01, you will have support for TLS 1.2. There are no special considerations for this package to support TLS 1.2 beyond that.
